fastlane/Fastfile

```diff
@@ -382,7 +382,7 @@ platform :ios do
     after_confirming_push(push_merge_branch: true) do
       trigger_beta_build(branch_to_build: release_branch_name)
 
-      copy_branch_protection(repository: GHHELPER_REPO, from_branch: DEFAULT_BRANCH, to_branch: release_branch_name)
+      set_branch_protection(repository: GHHELPER_REPO, branch: release_branch_name)
```

Comment:
@iangmaia should we add a comment as suggested by @AliSoftware [here](https://github.com/Automattic/pocket-casts-ios/pull/2050#issuecomment-2293557555)?

Reply:
Oh yes, I changed locally but didn't commit! 🤦‍♂️ 
Updated here: https://github.com/Automattic/pocket-casts-ios/pull/2052/commits/94f0d05708d3efabfe1dabe9c1bc5ed16e06f0b6